src/main/java/org/javarosa/core/util/DataUtil.java

```diff
@@ -67,6 +67,15 @@ public static String listToString(List<String> list) {
         return s.split("[ ]+");
     }
 
+    public static boolean intArrayContains(int[] source, int target) {
+        for (int sort: source) {
```

Comment:
This variable should be called something more generic instead of `sort` since it could be any int